aleph0/src/createActions.ts

```diff
@@ -0,0 +1,129 @@
+import { z } from "zod";
+import { loadSnippets } from "./lib/utils";
+import { createFile, createFolder, readFile } from "./lib/file";
+import { Technology } from "./types";
+import { RunnableFunctionWithParse } from "openai/lib/RunnableFunction.mjs";
+
+export const createActions = (options: {
+  technology: Technology;
+}): Record<string, RunnableFunctionWithParse<any>> => {
+  const { technology } = options;
+
+  return {
+    getSnippets: {
+      function: async (_args: {}) => {
+        return { snippets: await loadSnippets(technology) };
+      },
```

Comment:
would only make sense to ask the AI to call this for us if technology is an arg. otherwise we should just call it ourselves for the AI if we know it must be called each time regardless

Reply:
Good point, make the changes and push?

Reply:
TODO: remove this funcitoncall and just call once at the beginning. @elie222